Documentation/Hosting/AmazonGameLift/GameLift.md

```diff
@@ -0,0 +1,105 @@
+# MultiplayerSample Project with Amazon GameLift
+
+This README covers testing and running MultiplayerSample with Amazon GameLift.
+
+## Running with [Amazon GameLift](https://docs.aws.amazon.com/gamelift/index.html)
+
+### Setup
+1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html), if you don't already have it. You will need it to interact with Amazon GameLift.
+1. Enable the "AWSGameLift" and "MPSGameLift" gem by adding them to MultiplayerSample/Gem/Code/enabled_gems.cmake
+1. Build the server, game launchers, and asset bundler for MultiplayerSample as normal, per [top-level README](/README.md).
+    ```sh
+    cmake --build build\windows --target Editor MultiplayerSample.GameLauncher MultiplayerSample.ServerLauncher AssetBundler --config profile -- /m /nologo
+    ```
+1. Build all the assets
+    ```sh
+    cmake --build build\windows --target MultiplayerSample.Assets --config profile -- /m /nologo
+    ```
+1. Work in progress (WiP) step: Add your AWS region to Config/default_aws_resource_mappings.json (example: "Region": "us-west-2")
+    a. Currently needed otherwise when the client initializes GameLift there will be an error about not having a region. 
+    b. This step will be removed once we properly parse the game-session data which contains the fleet-id, region-id, etc  
+
+## Build Server for Windows (WiP)
+1. WiP Step: Build a profile pak server/game build
```

Comment:
Link to the these docs: https://github.com/o3de/o3de-multiplayersample/blob/development/Documentation/PackedAssetBuilds.md

Where we can, we should avoid content duplication.

Reply:
+1 on linking to the existing documents.

I do think it would be better however to use export-project python command that @tkothadev addedsupport to make it simpler for user to performs these steps.

Reply:
Spoke to @tkothadev, really cool stuff. I'll be implementing a script. Hoping to erase dozens of lines from this readme :)